prawcore/auth.py

```diff
@@ -167,6 +167,34 @@ def revoke(self, only_access=False):
             self.refresh_token = None
 
 
+class DeviceIDAuthorizer(Authorizer):
+    """Manages app-only OAuth2 for 'installed' applications.
+
+    While the '*' scope will be available, some endpoints simply will not work
+    due to the lack of an associated reddit account.
+    """
+
+    def __init__(self, authenticator, device_id):
```

Comment:
Do you think it makes sense to make DO_NOT_TRACK_THIS_DEVICE the default for the device_id param?